src/lib/InstantSearch.ts

```diff
@@ -133,23 +164,21 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
    * widget after InstantSearch started is considered **EXPERIMENTAL** and therefore
```

Comment:
We use and test those methods in the new major version. WDYT about removing the "experimental" mentions?

Reply:
agree, but can be done in another major version :)